package.json

```diff
@@ -9,7 +9,14 @@
     "_metalsmith": "bin/_metalsmith"
   },
   "scripts": {
-    "test": "make test"
+    "pretest": "npm install",
+    "test": "mocha $MOCHA_OPTS",
+    "pretest-debug": "npm install",
+    "test-debug": "mocha debug",
+    "precoverage": "npm install",
+    "coverage": "istanbul cover ./node_modules/.bin/_mocha",
```

Comment:
Looks like Istanbul isn't getting run with the generator flags on Node 0.12. Travis is erroring during this step: https://travis-ci.org/metalsmith/metalsmith/jobs/171375235